client/src/app/zeebe/Deployment.js

```diff
@@ -171,13 +170,9 @@ export default class Deployment extends EventEmitter {
     } = config;
 
     const {
-      id,
-      rememberCredentials
```

Comment:
I'm so happy to see `rememberCredentials` gone 😌. Missing the checkbox was a source of frustration.